controllers/machinehealthcheck_controller.go

```diff
@@ -256,7 +256,6 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 		)
 	}
 	for _, t := range healthy {
-		logger.V(3).Info("patching machine", "machine", t.Machine.GetName())
```

Comment:
After using MHC and tailing logs this just seems like an unnecessary log line, MHC is already pretty noisy and these patches are only on the *healthy* machines so are not especially important